pkg/cluster/internal/create/kubeadm-init.go

```diff
@@ -120,6 +120,40 @@ func runKubeadmInit(ec *execContext, configNode *NodeReplica) error {
 		return errors.Wrap(err, "failed to add default storage class")
 	}
 
+	// add dynamic volume provisioning
```

Comment:
/hold
I don't think this should be part of kubeadm init, and I think we should consider if we want to bundle this with the node image or not, and if installing this should be configurable.